src/vcpkg/vcpkgpaths.cpp

```diff
@@ -1332,20 +1332,14 @@ namespace vcpkg
                                  "",
                                  "Error: in triplet {triplet}: Unable to find a valid Visual Studio instance");
 
-    DECLARE_AND_REGISTER_MESSAGE(ErrorNoVSInstanceVersion,
-                                 (msg::version),
-                                 "Printed after ErrorNoVSInstance on a separate line",
-                                 "    with toolset version {version}");
+    DECLARE_AND_REGISTER_MESSAGE(ErrorNoVSInstanceVersion, (msg::version), "", "    with toolset version {version}");
 
     DECLARE_AND_REGISTER_MESSAGE(ErrorNoVSInstanceFullVersion,
                                  (msg::version),
-                                 "Printed after ErrorNoVSInstance on a separate line",
+                                 "",
                                  "    with toolset version prefix {version}");
 
-    DECLARE_AND_REGISTER_MESSAGE(ErrorNoVSInstanceAt,
-                                 (msg::path),
-                                 "Printed after ErrorNoVSInstance on a separate line",
-                                 "     at \"{path}\"");
+    DECLARE_AND_REGISTER_MESSAGE(ErrorNoVSInstanceAt, (msg::path), "", "     at \"{path}\"");
```

Comment:
As an aside, I still don't think formatting like indenting should be part of what translators see.

Reply:
we can fix this later.